src/app/_components/EnterCodeScreen/EnterCodeScreen.tsx

```diff
@@ -0,0 +1,29 @@
+import React from "react";
+import { Box, Typography, alpha } from "@mui/material";
+
+interface HaventGotEmailYetProps {
+  //Function to be called on Resend email click
+  onResendEmail: () => void;
+}
+
+const EnterCodeScreen = ({
+  onResendEmail,
+}: HaventGotEmailYetProps): JSX.Element => {
+  return (
+    <Box display="flex" flexDirection="row" gap="1">
+      <Typography id="havent-got-the-email-yet" color={alpha("#000000", 0.46)}>
```

Comment:
there is a space between the "Haven't got the email yet?" and "Resend email"

Reply:
that's right! I couldn't figure it out,  any idea how should I add that space?

Reply:
padding or margin bottom